WMass/python/plotter/plotUtils/utility.py

```diff
@@ -8,9 +8,19 @@
 from array import array
 import shutil
 from CMS_lumi import *
-    
+
 ROOT.gInterpreter.ProcessLine(".O3")
-ROOT.gInterpreter.ProcessLine('#include "ccFiles/systHistHelpers.cc"')
+#ROOT.gInterpreter.ProcessLine('#include "ccFiles/systHistHelpers.cc"')
```

Comment:
Maybe we should use gInterpreter.Declare (or if you prefer to compile the macro, best to delete rather than comment)

Reply:
Right, I could convert to Declare directly, after this morning discussion